conf/broker.conf

```diff
@@ -314,28 +311,19 @@ loadBalancerHostUsageCheckIntervalMinutes=1
 
 # Load shedding interval. Broker periodically checks whether some traffic should be offload from
 # some over-loaded broker to other under-loaded brokers
-loadBalancerSheddingIntervalMinutes=30
+loadBalancerSheddingIntervalMinutes=10
 
 # Prevent the same topics to be shed and moved to other broker more that once within this timeframe
-loadBalancerSheddingGracePeriodMinutes=30
-
-# Usage threshold to determine a broker as under-loaded
-loadBalancerBrokerUnderloadedThresholdPercentage=1
+loadBalancerSheddingGracePeriodMinutes=20
```

Comment:
I was thinking that a good default could be `sheddingInterval=5min` and `sheddingGracePeriod=30min` (since it's just to prevent from topic being bounced around many times)

Reply:
sure.